### ISSUE TITLE
unexpected duplicates when matching [Homo sapiens] against ott

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/nomer#127]
in discussion with @jar398 and @n8upham the following example was shown in attempt to align names with Open Tree of Life taxonomy, or at least Nomer's interpretation of it - 

```
echo -e "\tHomo sapiens" | nomer append ott
```

produced:

```
	Homo sapiens	HAS_ACCEPTED_NAME	OTT:770315	Homo sapiens		species		life | cellular organisms | Eukaryota | Opisthokonta | Holozoa | Metazoa | Eumetazoa | Bilateria | Deuterostomia | Chordata | Craniata | Vertebrata | Gnathostomata | Teleostomi | Euteleostomi | Sarcopterygii | Dipnotetrapodomorpha | Tetrapoda | Amniota | Mammalia | Theria | Eutheria | Boreoeutheria | Euarchontoglires | Primates | Haplorrhini | Simiiformes | Catarrhini | Hominoidea | Hominidae | Homininae | Homo | Homo sapiens	OTT:805080 | OTT:93302 | OTT:304358 | OTT:332573 | OTT:5246131 | OTT:691846 | OTT:641038 | OTT:117569 | OTT:147604 | OTT:125642 | OTT:947318 | OTT:801601 | OTT:278114 | OTT:114656 | OTT:114654 | OTT:458402 | OTT:4940726 | OTT:229562 | OTT:229560 | OTT:244265 | OTT:229558 | OTT:683263 | OTT:5334778 | OTT:392222 | OTT:913935 | OTT:702152 | OTT:386195 | OTT:842867 | OTT:386191 | OTT:770311 | OTT:312031 | OTT:770309 | OTT:770315	|  | domain |  |  | kingdom |  |  |  | phylum | subphylum | subphylum | superclass |  |  | superclass |  | superclass |  | class | subclass |  |  | superorder | order | suborder | infraorder | parvorder | superfamily | family | subfamily | genus | species		https://tree.opentreeoflife.org/opentree/ottol@770315
	Homo sapiens	SYNONYM_OF	OTT:770315	Homo sapiens		species		life | cellular organisms | Eukaryota | Opisthokonta | Holozoa | Metazoa | Eumetazoa | Bilateria | Deuterostomia | Chordata | Craniata | Vertebrata | Gnathostomata | Teleostomi | Euteleostomi | Sarcopterygii | Dipnotetrapodomorpha | Tetrapoda | Amniota | Mammalia | Theria | Eutheria | Boreoeutheria | Euarchontoglires | Primates | Haplorrhini | Simiiformes | Catarrhini | Hominoidea | Hominidae | Homininae | Homo | Homo sapiens	OTT:805080 | OTT:93302 | OTT:304358 | OTT:332573 | OTT:5246131 | OTT:691846 | OTT:641038 | OTT:117569 | OTT:147604 | OTT:125642 | OTT:947318 | OTT:801601 | OTT:278114 | OTT:114656 | OTT:114654 | OTT:458402 | OTT:4940726 | OTT:229562 | OTT:229560 | OTT:244265 | OTT:229558 | OTT:683263 | OTT:5334778 | OTT:392222 | OTT:913935 | OTT:702152 | OTT:386195 | OTT:842867 | OTT:386191 | OTT:770311 | OTT:312031 | OTT:770309 | OTT:770315	|  | domain |  |  | kingdom |  |  |  | phylum | subphylum | subphylum | superclass |  |  | superclass |  | superclass |  | class | subclass |  |  | superorder | order | suborder | infraorder | parvorder | superfamily | family | subfamily | genus | species		https://tree.opentreeoflife.org/opentree/ottol@770315
	Homo sapiens	SYNONYM_OF	OTT:770315	Homo sapiens		species		life | cellular organisms | Eukaryota | Opisthokonta | Holozoa | Metazoa | Eumetazoa | Bilateria | Deuterostomia | Chordata | Craniata | Vertebrata | Gnathostomata | Teleostomi | Euteleostomi | Sarcopterygii | Dipnotetrapodomorpha | Tetrapoda | Amniota | Mammalia | Theria | Eutheria | Boreoeutheria | Euarchontoglires | Primates | Haplorrhini | Simiiformes | Catarrhini | Hominoidea | Hominidae | Homininae | Homo | Homo sapiens	OTT:805080 | OTT:93302 | OTT:304358 | OTT:332573 | OTT:5246131 | OTT:691846 | OTT:641038 | OTT:117569 | OTT:147604 | OTT:125642 | OTT:947318 | OTT:801601 | OTT:278114 | OTT:114656 | OTT:114654 | OTT:458402 | OTT:4940726 | OTT:229562 | OTT:229560 | OTT:244265 | OTT:229558 | OTT:683263 | OTT:5334778 | OTT:392222 | OTT:913935 | OTT:702152 | OTT:386195 | OTT:842867 | OTT:386191 | OTT:770311 | OTT:312031 | OTT:770309 | OTT:770315	|  | domain |  |  | kingdom |  |  |  | phylum | subphylum | subphylum | superclass |  |  | superclass |  | superclass |  | class | subclass |  |  | superorder | order | suborder | infraorder | parvorder | superfamily | family | subfamily | genus | species		https://tree.opentreeoflife.org/opentree/ottol@770315
	Homo sapiens	SYNONYM_OF	OTT:770315	Homo sapiens		species		life | cellular organisms | Eukaryota | Opisthokonta | Holozoa | Metazoa | Eumetazoa | Bilateria | Deuterostomia | Chordata | Craniata | Vertebrata | Gnathostomata | Teleostomi | Euteleostomi | Sarcopterygii | Dipnotetrapodomorpha | Tetrapoda | Amniota | Mammalia | Theria | Eutheria | Boreoeutheria | Euarchontoglires | Primates | Haplorrhini | Simiiformes | Catarrhini | Hominoidea | Hominidae | Homininae | Homo | Homo sapiens	OTT:805080 | OTT:93302 | OTT:304358 | OTT:332573 | OTT:5246131 | OTT:691846 | OTT:641038 | OTT:117569 | OTT:147604 | OTT:125642 | OTT:947318 | OTT:801601 | OTT:278114 | OTT:114656 | OTT:114654 | OTT:458402 | OTT:4940726 | OTT:229562 | OTT:229560 | OTT:244265 | OTT:229558 | OTT:683263 | OTT:5334778 | OTT:392222 | OTT:913935 | OTT:702152 | OTT:386195 | OTT:842867 | OTT:386191 | OTT:770311 | OTT:312031 | OTT:770309 | OTT:770315	|  | domain |  |  | kingdom |  |  |  | phylum | subphylum | subphylum | superclass |  |  | superclass |  | superclass |  | class | subclass |  |  | superorder | order | suborder | infraorder | parvorder | superfamily | family | subfamily | genus | species		https://tree.opentreeoflife.org/opentree/ottol@770315
	Homo sapiens	SYNONYM_OF	OTT:770315	Homo sapiens		species		life | cellular organisms | Eukaryota | Opisthokonta | Holozoa | Metazoa | Eumetazoa | Bilateria | Deuterostomia | Chordata | Craniata | Vertebrata | Gnathostomata | Teleostomi | Euteleostomi | Sarcopterygii | Dipnotetrapodomorpha | Tetrapoda | Amniota | Mammalia | Theria | Eutheria | Boreoeutheria | Euarchontoglires | Primates | Haplorrhini | Simiiformes | Catarrhini | Hominoidea | Hominidae | Homininae | Homo | Homo sapiens	OTT:805080 | OTT:93302 | OTT:304358 | OTT:332573 | OTT:5246131 | OTT:691846 | OTT:641038 | OTT:117569 | OTT:147604 | OTT:125642 | OTT:947318 | OTT:801601 | OTT:278114 | OTT:114656 | OTT:114654 | OTT:458402 | OTT:4940726 | OTT:229562 | OTT:229560 | OTT:244265 | OTT:229558 | OTT:683263 | OTT:5334778 | OTT:392222 | OTT:913935 | OTT:702152 | OTT:386195 | OTT:842867 | OTT:386191 | OTT:770311 | OTT:312031 | OTT:770309 | OTT:770315	|  | domain |  |  | kingdom |  |  |  | phylum | subphylum | subphylum | superclass |  |  | superclass |  | superclass |  | class | subclass |  |  | superorder | order | suborder | infraorder | parvorder | superfamily | family | subfamily | genus | species		https://tree.opentreeoflife.org/opentree/ottol@770315

```

**Comment by jhpoelen** [comment]
see screenshot of related https://tree.opentreeoflife.org/opentree/opentree13.4@ott770315/Homo-sapiens below

![image](https://user-images.githubusercontent.com/1084872/202726440-97c5cbcb-53c3-45fa-a991-2ae57f07cd05.png)

**Comment by jhpoelen** [comment]
@jar398 turns out that I didn't complete the implementation of the OTT indexing after moving it from GloBI into Nomer. ; ( 

Good thing we stumbled across this, even though it is a little embarrassing to have this pop up in discussion with their creator . . . 

Hope to get to this sooner rather than later. OTT is such a rich resource for taxon linking, synonymy etc.

**Comment by jhpoelen** [comment]
After upgrading the opentree taxonomy support in nomer we found that:

```
$ echo -e "\tHomo sapiens" | nomer append ott
	Homo sapiens	HAS_ACCEPTED_NAME	OTT:770315	Homo sapiens		species		life | cellular organisms | Eukaryota | Opisthokonta | Holozoa | Metazoa | Eumetazoa | Bilateria | Deuterostomia | Chordata | Craniata | Vertebrata | Gnathostomata | Teleostomi | Euteleostomi | Sarcopterygii | Dipnotetrapodomorpha | Tetrapoda | Amniota | Mammalia | Theria | Eutheria | Boreoeutheria | Euarchontoglires | Primates | Haplorrhini | Simiiformes | Catarrhini | Hominoidea | Hominidae | Homininae | Homo | Homo sapiens	OTT:805080 | OTT:93302 | OTT:304358 | OTT:332573 | OTT:5246131 | OTT:691846 | OTT:641038 | OTT:117569 | OTT:147604 | OTT:125642 | OTT:947318 | OTT:801601 | OTT:278114 | OTT:114656 | OTT:114654 | OTT:458402 | OTT:4940726 | OTT:229562 | OTT:229560 | OTT:244265 | OTT:229558 | OTT:683263 | OTT:5334778 | OTT:392222 | OTT:913935 | OTT:702152 | OTT:386195 | OTT:842867 | OTT:386191 | OTT:770311 | OTT:312031 | OTT:770309 | OTT:770315	|  | domain |  |  | kingdom |  |  |  | phylum | subphylum | subphylum | superclass |  |  | superclass |  | superclass |  | class | subclass |  |  | superorder | order | suborder | infraorder | parvorder | superfamily | family | subfamily | genus | species		https://tree.opentreeoflife.org/opentree/ottol@770315
```

**Comment by jhpoelen** [comment]
After various updates, we now have re-introduced OTT mapping behaviors supporting name/external ids, and synonyms.

Synonym matching by name - 
```
$ echo -e "\tArius felis" | nomer append ott
	Arius felis	HAS_ACCEPTED_NAME	OTT:139650	Ariopsis felis		species		life | cellular organisms | Eukaryota | Opisthokonta | Holozoa | Metazoa | Eumetazoa | Bilateria | Deuterostomia | Chordata | Craniata | Vertebrata | Gnathostomata | Teleostomi | Euteleostomi | Actinopterygii | Actinopteri | Neopterygii | Teleostei | Osteoglossocephalai | Clupeocephala | Otomorpha | Ostariophysi | Otophysi | Characiphysae | Siluriformes | Siluroidei | Ariidae | Ariopsis | Ariopsis felis	OTT:805080 | OTT:93302 | OTT:304358 | OTT:332573 | OTT:5246131 | OTT:691846 | OTT:641038 | OTT:117569 | OTT:147604 | OTT:125642 | OTT:947318 | OTT:801601 | OTT:278114 | OTT:114656 | OTT:114654 | OTT:773483 | OTT:285821 | OTT:471203 | OTT:212201 | OTT:5506109 | OTT:285819 | OTT:761450 | OTT:803544 | OTT:285818 | OTT:285825 | OTT:701516 | OTT:5548749 | OTT:609233 | OTT:704260 | OTT:139650	|  | domain |  |  | kingdom |  |  |  | phylum | subphylum | subphylum | superclass |  |  | superclass | class | subclass | infraclass |  |  | cohort | subcohort |  | superorder | order | suborder | family | genus | species		https://tree.opentreeoflife.org/opentree/ottol@139650
```


Matching by NCBI id 
```
$ echo -e "NCBI:9606\t" | nomer append ott
NCBI:9606		HAS_ACCEPTED_NAME	OTT:770315	Homo sapiens		species		life | cellular organisms | Eukaryota | Opisthokonta | Holozoa | Metazoa | Eumetazoa | Bilateria | Deuterostomia | Chordata | Craniata | Vertebrata | Gnathostomata | Teleostomi | Euteleostomi | Sarcopterygii | Dipnotetrapodomorpha | Tetrapoda | Amniota | Mammalia | Theria | Eutheria | Boreoeutheria | Euarchontoglires | Primates | Haplorrhini | Simiiformes | Catarrhini | Hominoidea | Hominidae | Homininae | Homo | Homo sapiens	OTT:805080 | OTT:93302 | OTT:304358 | OTT:332573 | OTT:5246131 | OTT:691846 | OTT:641038 | OTT:117569 | OTT:147604 | OTT:125642 | OTT:947318 | OTT:801601 | OTT:278114 | OTT:114656 | OTT:114654 | OTT:458402 | OTT:4940726 | OTT:229562 | OTT:229560 | OTT:244265 | OTT:229558 | OTT:683263 | OTT:5334778 | OTT:392222 | OTT:913935 | OTT:702152 | OTT:386195 | OTT:842867 | OTT:386191 | OTT:770311 | OTT:312031 | OTT:770309 | OTT:770315	|  | domain |  |  | kingdom |  |  |  | phylum | subphylum | subphylum | superclass |  |  | superclass |  | superclass |  | class | subclass |  |  | superorder | order | suborder | infraorder | parvorder | superfamily | family | subfamily | genus | species		https://tree.opentreeoflife.org/opentree/ottol@770315
```

and listing all indexed OTT relations:

```
$ nomer ls ott | head
OTT:2	Elaeocarpus williamsianus		HAS_ACCEPTED_NAME	OTT:2	Elaeocarpus williamsianus		species		life | cellular organisms | Eukaryota | Archaeplastida | Chloroplastida | Streptophyta | Embryophyta | Tracheophyta | Euphyllophyta | Spermatophyta | Magnoliopsida | Mesangiospermae | eudicotyledons | Gunneridae | Pentapetalae | rosids | fabids | Oxalidales | Elaeocarpaceae | Elaeocarpus | Elaeocarpus williamsianus	OTT:805080 | OTT:93302 | OTT:304358 | OTT:5268475 | OTT:361838 | OTT:916750 | OTT:56610 | OTT:10210 | OTT:1007992 | OTT:10218 | OTT:99252 | OTT:5298374 | OTT:431495 | OTT:853757 | OTT:5316182 | OTT:1008296 | OTT:565281 | OTT:431493 | OTT:649007 | OTT:842169 | OTT:2	|  | domain | kingdom | kingdom | phylum |  | phylum |  |  | class |  |  |  |  |  |  | order | family | genus | species		https://tree.opentreeoflife.org/opentree/ottol@2
OTT:4	Platytheca galioides		SYNONYM_OF	OTT:4	Platytheca galioides		species		life | cellular organisms | Eukaryota | Archaeplastida | Chloroplastida | Streptophyta | Embryophyta | Tracheophyta | Euphyllophyta | Spermatophyta | Magnoliopsida | Mesangiospermae | eudicotyledons | Gunneridae | Pentapetalae | rosids | fabids | Oxalidales | Elaeocarpaceae | Platytheca | Platytheca galioides	OTT:805080 | OTT:93302 | OTT:304358 | OTT:5268475 | OTT:361838 | OTT:916750 | OTT:56610 | OTT:10210 | OTT:1007992 | OTT:10218 | OTT:99252 | OTT:5298374 | OTT:431495 | OTT:853757 | OTT:5316182 | OTT:1008296 | OTT:565281 | OTT:431493 | OTT:649007 | OTT:414967 | OTT:4	|  | domain | kingdom | kingdom | phylum |  | phylum |  |  | class |  |  |  |  |  |  | order | family | genus | species		https://tree.opentreeoflife.org/opentree/ottol@4
OTT:6	Elaeocarpus foveolatus		HAS_ACCEPTED_NAME	OTT:6	Elaeocarpus foveolatus		species		life | cellular organisms | Eukaryota | Archaeplastida | Chloroplastida | Streptophyta | Embryophyta | Tracheophyta | Euphyllophyta | Spermatophyta | Magnoliopsida | Mesangiospermae | eudicotyledons | Gunneridae | Pentapetalae | rosids | fabids | Oxalidales | Elaeocarpaceae | Elaeocarpus | Elaeocarpus foveolatus	OTT:805080 | OTT:93302 | OTT:304358 | OTT:5268475 | OTT:361838 | OTT:916750 | OTT:56610 | OTT:10210 | OTT:1007992 | OTT:10218 | OTT:99252 | OTT:5298374 | OTT:431495 | OTT:853757 | OTT:5316182 | OTT:1008296 | OTT:565281 | OTT:431493 | OTT:649007 | OTT:842169 | OTT:6	|  | domain | kingdom | kingdom | phylum |  | phylum |  |  | class |  |  |  |  |  |  | order | family | genus | species		https://tree.opentreeoflife.org/opentree/ottol@6
OTT:8	Elaeocarpus sericopetalus		HAS_ACCEPTED_NAME	OTT:8	Elaeocarpus sericopetalus		species		life | cellular organisms | Eukaryota | Archaeplastida | Chloroplastida | Streptophyta | Embryophyta | Tracheophyta | Euphyllophyta | Spermatophyta | Magnoliopsida | Mesangiospermae | eudicotyledons | Gunneridae | Pentapetalae | rosids | fabids | Oxalidales | Elaeocarpaceae | Elaeocarpus | Elaeocarpus sericopetalus	OTT:805080 | OTT:93302 | OTT:304358 | OTT:5268475 | OTT:361838 | OTT:916750 | OTT:56610 | OTT:10210 | OTT:1007992 | OTT:10218 | OTT:99252 | OTT:5298374 | OTT:431495 | OTT:853757 | OTT:5316182 | OTT:1008296 | OTT:565281 | OTT:431493 | OTT:649007 | OTT:842169 | OTT:8	|  | domain | kingdom | kingdom | phylum |  | phylum |  |  | class |  |  |  |  |  |  | order | family | genus | species		https://tree.opentreeoflife.org/opentree/ottol@8
OTT:10	Elaeocarpus culminicola		SYNONYM_OF	OTT:10	Elaeocarpus culminicola		species		life | cellular organisms | Eukaryota | Archaeplastida | Chloroplastida | Streptophyta | Embryophyta | Tracheophyta | Euphyllophyta | Spermatophyta | Magnoliopsida | Mesangiospermae | eudicotyledons | Gunneridae | Pentapetalae | rosids | fabids | Oxalidales | Elaeocarpaceae | Elaeocarpus | Elaeocarpus culminicola	OTT:805080 | OTT:93302 | OTT:304358 | OTT:5268475 | OTT:361838 | OTT:916750 | OTT:56610 | OTT:10210 | OTT:1007992 | OTT:10218 | OTT:99252 | OTT:5298374 | OTT:431495 | OTT:853757 | OTT:5316182 | OTT:1008296 | OTT:565281 | OTT:431493 | OTT:649007 | OTT:842169 | OTT:10	|  | domain | kingdom | kingdom | phylum |  | phylum |  |  | class |  |  |  |  |  |  | order | family | genus | species		https://tree.opentreeoflife.org/opentree/ottol@10
OTT:12	Elaeocarpus arnhemicus		SYNONYM_OF	OTT:12	Elaeocarpus arnhemicus		species		life | cellular organisms | Eukaryota | Archaeplastida | Chloroplastida | Streptophyta | Embryophyta | Tracheophyta | Euphyllophyta | Spermatophyta | Magnoliopsida | Mesangiospermae | eudicotyledons | Gunneridae | Pentapetalae | rosids | fabids | Oxalidales | Elaeocarpaceae | Elaeocarpus | Elaeocarpus arnhemicus	OTT:805080 | OTT:93302 | OTT:304358 | OTT:5268475 | OTT:361838 | OTT:916750 | OTT:56610 | OTT:10210 | OTT:1007992 | OTT:10218 | OTT:99252 | OTT:5298374 | OTT:431495 | OTT:853757 | OTT:5316182 | OTT:1008296 | OTT:565281 | OTT:431493 | OTT:649007 | OTT:842169 | OTT:12	|  | domain | kingdom | kingdom | phylum |  | phylum |  |  | class |  |  |  |  |  |  | order | family | genus | species		https://tree.opentreeoflife.org/opentree/ottol@12
OTT:14	Elaeocarpus hookerianus		HAS_ACCEPTED_NAME	OTT:14	Elaeocarpus hookerianus		species		life | cellular organisms | Eukaryota | Archaeplastida | Chloroplastida | Streptophyta | Embryophyta | Tracheophyta | Euphyllophyta | Spermatophyta | Magnoliopsida | Mesangiospermae | eudicotyledons | Gunneridae | Pentapetalae | rosids | fabids | Oxalidales | Elaeocarpaceae | Elaeocarpus | Elaeocarpus hookerianus	OTT:805080 | OTT:93302 | OTT:304358 | OTT:5268475 | OTT:361838 | OTT:916750 | OTT:56610 | OTT:10210 | OTT:1007992 | OTT:10218 | OTT:99252 | OTT:5298374 | OTT:431495 | OTT:853757 | OTT:5316182 | OTT:1008296 | OTT:565281 | OTT:431493 | OTT:649007 | OTT:842169 | OTT:14	|  | domain | kingdom | kingdom | phylum |  | phylum |  |  | class |  |  |  |  |  |  | order | family | genus | species		https://tree.opentreeoflife.org/opentree/ottol@14
OTT:16	Elaeocarpus kirtonii		SYNONYM_OF	OTT:16	Elaeocarpus kirtonii		species		life | cellular organisms | Eukaryota | Archaeplastida | Chloroplastida | Streptophyta | Embryophyta | Tracheophyta | Euphyllophyta | Spermatophyta | Magnoliopsida | Mesangiospermae | eudicotyledons | Gunneridae | Pentapetalae | rosids | fabids | Oxalidales | Elaeocarpaceae | Elaeocarpus | Elaeocarpus kirtonii	OTT:805080 | OTT:93302 | OTT:304358 | OTT:5268475 | OTT:361838 | OTT:916750 | OTT:56610 | OTT:10210 | OTT:1007992 | OTT:10218 | OTT:99252 | OTT:5298374 | OTT:431495 | OTT:853757 | OTT:5316182 | OTT:1008296 | OTT:565281 | OTT:431493 | OTT:649007 | OTT:842169 | OTT:16	|  | domain | kingdom | kingdom | phylum |  | phylum |  |  | class |  |  |  |  |  |  | order | family | genus | species		https://tree.opentreeoflife.org/opentree/ottol@16
OTT:20	Tetratheca filiformis		HAS_ACCEPTED_NAME	OTT:20	Tetratheca filiformis		species		life | cellular organisms | Eukaryota | Archaeplastida | Chloroplastida | Streptophyta | Embryophyta | Tracheophyta | Euphyllophyta | Spermatophyta | Magnoliopsida | Mesangiospermae | eudicotyledons | Gunneridae | Pentapetalae | rosids | fabids | Oxalidales | Elaeocarpaceae | Tetratheca | Tetratheca filiformis	OTT:805080 | OTT:93302 | OTT:304358 | OTT:5268475 | OTT:361838 | OTT:916750 | OTT:56610 | OTT:10210 | OTT:1007992 | OTT:10218 | OTT:99252 | OTT:5298374 | OTT:431495 | OTT:853757 | OTT:5316182 | OTT:1008296 | OTT:565281 | OTT:431493 | OTT:649007 | OTT:730351 | OTT:20	|  | domain | kingdom | kingdom | phylum |  | phylum |  |  | class |  |  |  |  |  |  | order | family | genus | species		https://tree.opentreeoflife.org/opentree/ottol@20
OTT:22	Tetratheca juncea		SYNONYM_OF	OTT:22	Tetratheca juncea		species		life | cellular organisms | Eukaryota | Archaeplastida | Chloroplastida | Streptophyta | Embryophyta | Tracheophyta | Euphyllophyta | Spermatophyta | Magnoliopsida | Mesangiospermae | eudicotyledons | Gunneridae | Pentapetalae | rosids | fabids | Oxalidales | Elaeocarpaceae | Tetratheca | Tetratheca junceaOTT:805080 | OTT:93302 | OTT:304358 | OTT:5268475 | OTT:361838 | OTT:916750 | OTT:56610 | OTT:10210 | OTT:1007992 | OTT:10218 | OTT:99252 | OTT:5298374 | OTT:431495 | OTT:853757 | OTT:5316182 | OTT:1008296 | OTT:565281 | OTT:431493 | OTT:649007 | OTT:730351 | OTT:22	|  | domain | kingdom | kingdom | phylum |  | phylum |  |  | class |  |  |  |  |  |  | order | family | genus | species		https://tree.opentreeoflife.org/opentree/ottol@22
```

**Comment by jhpoelen** [comment]
ott indexing now available in v0.4.5
